Makefile

```diff
@@ -9,7 +9,7 @@ endif
 GDB=-ggdb3
 LIBS=-lm -lsdpa -llapack -lblas -lgfortran -lquadmath -lpthread -ldmumps_seq
 STATIC=-static-libgcc -static-libstdc++
-WARNS=-Wextra -Wall -Wno-sign-compare -Wshadow -Wstrict-aliasing=1 -Werror -pedantic-errors
+WARNS=-Wextra -Wall -Wno-sign-compare -Wshadow -Wstrict-aliasing=1 -pedantic-errors
```

Comment:
Thanks for the reply.

With the error `-Werror` flag it would not compile.

![werror](https://user-images.githubusercontent.com/5663725/38375383-5a38987a-38f6-11e8-9bd7-507ae57a37de.png)